tests/testsuite/login.rs

```diff
@@ -126,6 +126,38 @@ fn empty_login_token() {
         .run();
 }
 
+#[cargo_test]
+fn invalid_login_token() {
+    let registry = RegistryBuilder::new()
+        .no_configure_registry()
+        .no_configure_token()
+        .build();
+    setup_new_credentials();
+
+    let check_ = |stdin: &str, stderr: &str| {
+        cargo_process("login")
+            .replace_crates_io(registry.index_url())
+            .with_stdout("please paste the token found on [..]/me below")
+            .with_stdin(stdin)
+            .with_stderr(stderr)
+            .with_status(101)
+            .run();
+    };
+    let check = |stdin: &str| {
+        check_(stdin, "[ERROR] invalid token.");
```

Comment:
I could go either way on whether this helper function pulls its weight.
I'm leaning toward just using the two argument version even though it means repeating `"[ERROR] invalid token."`. But certainly could be convinced otherwise.